Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -50,7 +50,7 @@ def tap_new
       on:
         push:
           branches: master
-        pull_request: []
+        pull_request: ~
```

Comment:
```suggestion
        pull_request:
```

Reply:
I believe that might have a problem, I think that would translate to an empty string

We could go with `null` though, that would be the same

Reply:
Ah no! I'm wrong, that's changed with the 1.2 version of yaml! (You learn something new every day)